cmd/osm-bootstrap/osm-bootstrap.go

```diff
@@ -404,6 +404,15 @@ func (b *bootstrap) createMeshRootCertificate() error {
 		return err
 	}
 
+	createdMRC.Status = configv1alpha2.MeshRootCertificateStatus{
+		State: constants.MRCStateActive,
+	}
+
+	_, err = b.configClient.ConfigV1alpha2().MeshRootCertificates(b.namespace).UpdateStatus(context.Background(), createdMRC, metav1.UpdateOptions{})
+	if err != nil {
```

Comment:
If the err indicates that the attempted update conflicts with the existing resource, I don't think an error should be returned. If there are multiple instances of the osm-bootstrap this could happen. In this case, we can log an error or warning

Reply:
Ah, thank you; good callout!